src/util/chalk.js

```diff
@@ -0,0 +1,7 @@
+/* @flow */
+
+const chalk = require('chalk');
+
+chalk.enabled = process.stdout.isTTY;
```

Comment:
Chalk already handles this: https://github.com/chalk/supports-color/blob/8400d98ade32b2adffd50902c06d9e725a5c6588/index.js#L29